src/Transforms/Tools/MeasureEntropy.cs

```diff
@@ -43,9 +43,9 @@ await foreach (var block in input)
                     .Select(x => - Math.Log2(x) * x)
                     .Sum();
                 var size = block.Dictionary.Serialize().Length;
-                Console.WriteLine($"this many symbols {block.Stream.Length}");
+                Console.WriteLine($"Number of symbols: {block.Stream.Length}");
                 Console.WriteLine($"[{this.GetHashCode()}] Entropy: {entropy}; Count: {this.ODInt.Count}");
-                Console.WriteLine($"dictionary takes {size} bytes");
+                Console.WriteLine($"Dictionary size: {size} bytes");
```

Comment:
It'd be awesome if we estimate the dictionary space with its entropy as well. Very soon there won't be an s token, as words will be encoded using an entropy coder and we won't be limited by the dictionary size. But for now, the s token takes the most space and it's mostly separator characters `0xff`